build1.sh

```diff
@@ -38,6 +38,9 @@ chmod +x meilix-generator.sh
 echo "$(<meilix-generator.sh)" #to test the file
 ./meilix-generator.sh
 rm meilix-generator.sh
+
+#change host name
```

Comment:
This may solve the #104 ?
or consider including this all https://github.com/fossasia/meilix-generator/issues/79#issuecomment-317121343